NestedAccordion/js/main.js

```diff
@@ -0,0 +1,20 @@
+$('section').hide(); //hides entire section element at first.
+
+
+$('h3').on('click', function() {
+  if ($(this).next().is(':hidden')) { //if the next tag(section) after the current element (h3) is hidden....(:hidden is a visibility filter)
```

Comment:
using lots of `this` -- where'd you find out about `this`? I'd love to see your source. :)